src/main/java/com/sublinks/sublinksapi/queue/QueueConfig.java

```diff
@@ -9,28 +9,31 @@
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class QueueConfig {
-  @Value("${BACKEND_QUEUE_NAME}")
+  @Value("${BACKEND_QUEUE_NAME:}")
```

Comment:
Are those then ENV variables? Why not application properties?

Reply:
I can change 👍

Reply:
We would preferably be all in the application properties. Just to sort them all in one place and not spread small crumbs of env there and there.